python/vowpalwabbit/pyvw.py

```diff
@@ -1458,7 +1441,7 @@ def __init__(self, colname, expected_type, min_value=None):
 
         Parameters
         ----------
-        colname: str/int/float
+        colname: any hashable type (str/int/float/tuple/etc.)
```

Comment:
not actually link to the current PR. It's just that pandas accept all these types as colname (same as allowed keys for dict type I think).

Reply:
do you think it's worthwhile adding that as a comment in the code itself to avoid confusion with vw format, etc

Reply:
I think I could specify the type directly as  `colname: any valid pandas's column name (str/int/float/etc.)`

Reply:
agree with being more explicit mentioning panda's here